### ISSUE TITLE
Automate the generation of files on modification

### ISSUE THREAD

**Issue by maxime645** [TypeCobolTeam/TypeCobol#1211]
Currently, we have some files that need to be generated manually (e.g. Cup files or skeletons in PR #1178 )
We would like to automate the process when needed, for instance when skeletons.xml is modified.

Some tools like [CustomTools](https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/custom-tools?view=vs-2017) might help. Exemple of implementation: https://github.com/thomaslevesque/AutoRunCustomTool

**Comment by mayanje** [comment]
Yes this mechanism needs to install a VS extension, another way to do that is to do what ANTLR does for its .g4 extension files, ANTLR is based on nuget and uses Microsoft.Build.Utilities.Task directly related to an MSBuild concept.
https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.utilities.task?view=netframework-4.7.2
